src/Sylius/Component/User/Context/CustomerContextInterface.php

```diff
@@ -19,7 +19,7 @@
 interface CustomerContextInterface
 {
     /**
-     * @return CustomerInterface|null
+     * @return null|CustomerInterface
```

Comment:
So if there is no sense in this interface, why do you even bother about it? :) And more, why do you change senseless phpdoc order in senseless interface? 
BTW, closing a discussion in such a rude manner, you make reviewing your PRs really *senseless* ;)